Dfe.Academies.Academisation.Domain.Core/ApplicationAggregate/SchoolDetails.cs

```diff
@@ -19,6 +19,7 @@ public record SchoolDetails(
 	string? ApproverContactName = null,
 	string? ApproverContactEmail = null,
 	// conversion details
+	bool? SchoolConversionTargetDateSpecified = null,
```

Comment:
One last tiny edit, could we drop "School" from the name and just call it `ConversionTargetDateSpecified`? Then it's consistent with the other fields.